src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -30,36 +30,7 @@ import { analyticsV2 } from '@/analytics';
 import { divWorklet, equalWorklet, greaterThanWorklet, isNumberStringWorklet, mulWorklet } from '@/__swaps__/safe-math/SafeMath';
 import { SPRING_CONFIGS } from '@/components/animations/animationConfigs';
 import { triggerHapticFeedback } from '@/screens/points/constants';
-
-function getInputValuesForSliderPositionWorklet({
```

Comment:
I just moved this out to a separate file to make it easier for testing later - no changes